components/ProfileCard.tsx

```diff
@@ -3,14 +3,29 @@ import LinkedInIcon from "@icons/linkedin.svg";
 import DiscordIcon from "@icons/discord.svg";
 import Image from "next/image";
 
-export const ProfileCard = ({ profile }) => {
-  const socialIconMap = {
+type SocialType = "discord" | "instagram" | "linkedIn";
+
+export interface Profile {
+  id: string;
```

Comment:
You don't need `id` for this, since the listings in `exec.json` does not have the `id` fields